doc/cephfs/add-remove-mds.rst

```diff
@@ -62,7 +62,15 @@ means limiting its cache size.
 Adding an MDS
 =============
 
-#. Create an mds data point ``/var/lib/ceph/mds/ceph-${id}``. The daemon only uses this directory to store its keyring.
+NOTE: It is highly recommended to use `cephadm`_ for setting up the ceph
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+cluster and thus the MDS(s). Use this approach only if you are setting up the
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+ceph cluster manually. If one still intends to use the manual way for deploying
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+MDS daemons, `ceph orchestrator`_ can also be used.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Question: do the carats (^) do some sort of markdown thing?

Reply:
Yes, As it was decided that we need to provide a note that is fairly visible, using carats (^) over here would give us a bold text with a big font size that will be clearly visible. Although the size would be smaller than the section heading so it won't look odd at all.